samples/android/Continote/app/src/main/java/com/firebasecontinue/sample/continote/MainActivity.java

```diff
@@ -46,43 +48,52 @@
 
     // UI elements
     @Nullable
-    private TextView authMessageTextView = null;
+    private TextView mAuthMessageTextView = null;
     @Nullable
-    private Button authButton = null;
+    private Button mAuthButton = null;
     @Nullable
-    private Button myNotesButton = null;
+    private Button mMyNotesButton = null;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         setContentView(R.layout.activity_main);
 
+        // Force Facebook authentication to use the web. This simplifies the setup process for this
+        // sample since it requires the minimal amount of configuration within Facebook.
+        LoginManager.getInstance().setLoginBehavior(LoginBehavior.WEB_ONLY);
+
         // Gather the UI elements for this Activity for future manipulation.
-        authMessageTextView = (TextView) findViewById(R.id.authMessageTextView);
-        authButton = (Button) findViewById(R.id.authButton);
-        myNotesButton = (Button) findViewById(R.id.myNotesButton);
+        mAuthMessageTextView = (TextView) findViewById(R.id.authMessageTextView);
+        mAuthButton = (Button) findViewById(R.id.authButton);
+        mMyNotesButton = (Button) findViewById(R.id.myNotesButton);
     }
 
     @Override
     protected void handleUserSignedIn(@NonNull FirebaseUser user) {
         super.handleUserSignedIn(user);
 
+        if (user == null) {
+            // This should never happen, but just in case.
+            throw new AssertionError("user must be non-null");
+        }
+
         // Update the UI to reflect the user now being signed in.
 
-        if (authMessageTextView != null) {
-            authMessageTextView.setText(
+        if (mAuthMessageTextView != null) {
```

Comment:
Why do you have all these null checks? They really _should_ all exist since you're intializing them all during onCreate().

If you're concerned about onCreate() not working properly, I'd verify their existences all there with Assertions.